.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: go
 
 go:
-  - 1.8.3
+  - "1.10"
```

Comment:
Can we add a 1.9.X version? I think we should keep testing whatever version is used upstream.